src/lightning_app/cli/lightning_cli_create.py

```diff
@@ -54,9 +54,8 @@ def create() -> None:
     "do_async",
     type=bool,
     required=False,
-    default=False,
-    is_flag=True,
```

Comment:
it probably means the opposite is needed

Reply:
yup, flipped it around :+1: